src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -355,13 +355,14 @@ namespace winrt::Microsoft::Terminal::Control::implementation
                 return drawableDataStart + stride * y;
             };
             // A helper to draw a single pip (mark) at the given location.
-            const auto drawPip = [&](uint8_t* beg, til::color color) [[msvc::forceinline]] {
+            const auto drawPip = [&](uint8_t* beg, COLORREF color) [[msvc::forceinline]] {
                 const auto end = beg + pipHeight * stride;
                 for (; beg < end; beg += stride)
                 {
-                    // Coincidentally a til::color has the same RGBA format as the bitmap.
+                    // a til::color does NOT have the same RGBA format as the bitmap.
 #pragma warning(suppress : 26490) // Don't use reinterpret_cast (type.1).
-                    std::fill_n(reinterpret_cast<til::color*>(beg), pipWidth, color);
+                    const DWORD c = 0xff << 24 | GetRValue(color) << 16 | GetGValue(color) << 8 | GetBValue(color);

```

Comment:
if you're going to shuffle the bits around manually, why change it to `COLORREF` at all? Just use `til::color::r`, `til::color::g`, etc.

Reply:
Sorry for the testing-fuckup. I concur with Dustin. The `beg` pointer is already a `uint8_t*` so we can just:
```cpp
*beg++ = color.r;
*beg++ = color.g;
*beg++ = color.b;
*beg++ = color.a;
```
(Or the other way around?)